lib/routemaster/drain.rb

```diff
@@ -1,5 +1,5 @@
 module Routemaster
   module Drain
-    VERSION = '3.6.4'.freeze
+    VERSION = '3.6.5'.freeze
```

Comment:
please exclude version updates in your change. simply add your changes to the CHANGELOG file. we'll then do a separate gem bumping update like here:

https://github.com/deliveroo/routemaster-drain/commit/575569e6a8784fcde0884e4070cc6e64d5e1e058